content/en/docs/reference/access-authn-authz/webhook.md

```diff
@@ -121,6 +121,24 @@ To disallow access, the remote service would return:
 }
 ```
 
+A special case is when an authorization webhook can't make a decision based on
+the information available. By setting both status attributes `allowed` and `denied`
+to `false` the webhook signals "no opinion",  which will forward the request to the
+next configured authorizer. This is useful when using webhook authorization for
+certain types of requests (like regular users) and another type of authorization for
+other type of users (such as service accounts).
+
+```json
+{
+  "apiVersion": "authorization.k8s.io/v1beta1",
+  "kind": "SubjectAccessReview",
+  "status": {
+    "allowed": false,
+    "denied": false
```

Comment:
This is equivalent to the example above on line 111, since the absent `denied` attribute there is interpreted as "false"